src/main/java/com/iexec/common/chain/DealParams.java

```diff
@@ -17,11 +17,69 @@
 package com.iexec.common.chain;
 
 import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
 import lombok.*;
+import lombok.extern.slf4j.Slf4j;
 
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Collections;
 import java.util.List;
 import java.util.Map;
 
+import static com.fasterxml.jackson.annotation.JsonInclude.Include.*;
+
+/**
+ * The DealParams class represents runtime configurations provided for tasks computation.
+ * DealParams allow a requester to provide <b>args</b>, <b>input files</b>, <b>secrets</b>,
+ * <b>result storage configuration</b>... Those configurations are written on the blockchain
+ * when a deal is submitted.
+ * <p>
```

Comment:
I usually prefer to use `<br>` tag. Don't know what @jeremyjams and @zguesmi prefer.

Reply:
b stands for bold :)

Reply:
I followed this documentation https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html
I like `<p>` which stands for paragraph but `<br>` is fine as well.

Reply:
OK comment was on `<p>` ..!

Default Intellij likes `<p>` too:
![image](https://user-images.githubusercontent.com/33313130/149810600-d9c1154e-3891-4f60-a8ba-a4eb31fc26bc.png)

Reply:
If both of you prefer `<p>`, let's use that tag so we're consistent :)